profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -223,6 +223,110 @@ function nexteuropa_varnish_node_presave($node) {
   }
 }
 
+/**
+ * Implements hook_node_delete().
+ */
+function nexteuropa_varnish_node_delete($node) {
+  // No rule defined for the node: Ignore the rest of the process.
+  if (!nexteuropa_varnish_node_has_rules($node)) {
+    return;
+  }
+
+  $paths = array();
+
+  $entity_uri = entity_uri('node', $node);
+
+  if ($entity_uri) {
```

Comment:
Just for the sake of maintainability in the future it would be nice to avoid this kind of "ghosts" `if` conditions.
I don't say it's wrong, but as PHP is not strongly typed (yet?) and even less Drupal, it would be easier for everyone to have a better condition there.

From the API documentation, [entity_uri()](https://api.drupal.org/api/drupal/includes%21common.inc/function/entity_uri/7.x) returns:
_An array containing the 'path' and 'options' keys used to build the URI of the entity, and matching the signature of url() or NULL if the entity has no URI of its own._

Maybe the proper condition here would have been:

`if (null !== $entity_uri)` or `if (is_array($entity_uri))`, with a preference for the first one, which is faster.

Reply:
Fixed.